spec/models/like_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before(:example) do
+    @user = User.new(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Poland.')
+    @post = Post.new(author: @user, title: 'Hello', text: 'This is my first post')
+  end
+
+  subject { Like.new(user: @user, post: @post) }
+
+  it 'name should be present' do
+    subject.user = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should be present' do
+    subject.post = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Great job working on this milestone. Your unit tests for the Like model are failing. This is because you are writing tests for name should be present and title should be present validations which are not present in the like.rb model file. I think you should rather write unit test for the method found in the Like model.
![image](https://github.com/MMhaidari/blog-app/assets/62849872/7568a775-beca-4f57-8f52-657a483852e1)